osu.Game/Overlays/Settings/Sections/SkinSection.cs

```diff
@@ -81,6 +102,9 @@ private void load(OsuConfigManager config)
             skinDropdown.Bindable = dropdownBindable;
             skinDropdown.Items = skins.GetAllUsableSkins().ToArray();
 
+            currentSkin = skins.CurrentSkin.GetBoundCopy();
+            currentSkin.BindValueChanged(skin => exportButton.Enabled.Value = skin.NewValue.SkinInfo.ID > 0);
```

Comment:
Should check immediately:
```suggestion
            currentSkin.BindValueChanged(skin => exportButton.Enabled.Value = skin.NewValue.SkinInfo.ID > 0, true);
```